### ISSUE TITLE
[BUG] `xcparse` not working with Swift 5.4

### ISSUE THREAD

**Issue by nachocual** [ChargePoint/xcparse#67]
**Describe the bug**
The latest version of `xcparse`, 2.2, doesn't work with Swift 5.4

**Desktop**
 - OS: macOS 11.5
 - xcparse version: 2.2
 - Swift version 5.4.2

**To Reproduce**
1. Install the latest version of xcparse using Homebrew.
2. Run it.
4. See error:
    ```
    dyld: Library not loaded: @rpath/libSwiftToolsSupport.dylib
      Referenced from: /usr/local/bin/xcparse
      Reason: image not found
    ```

**Comment by nachocual** [comment]
I might be completely off, but you might need to change `SwiftToolsSupport` to `SwiftToolsSupport-auto` in  you package definition from what I saw [here](https://developer.apple.com/forums/thread/655937).

**Comment by andrewclaus** [comment]
Also seeing this same issue, started after the 2.2 release was created a few hours ago.

**Comment by abotkin-cpi** [comment]
Yep, making the fix now; really need to get this put on the new CI/CD we have.

**Comment by abotkin-cpi** [comment]
This should be resolved with v2.2.1. Thanks @nachocual for the details!

**Comment by nachocual** [comment]
Thanks for fixing it that quickly!
